.travis.yml

```diff
@@ -48,11 +48,10 @@ script:
   - go vet ./...
     # For compatibility with Go 1.5, launch only if megacheck is present,
     # ignore SA1019 (deprecation warnings) in conn_test.go (we have to use the
-    # deprecated driver.Execer and driver.Queryer interfaces) and S1024
-    # (time.Until) everywhere.
+    # deprecated driver.Execer and driver.Queryer interfaces).
   - >
     which megacheck > /dev/null
-    && megacheck -ignore 'github.com/lib/pq/conn_test.go:SA1019 github.com/lib/pq/*.go:S1024' ./...
+    && megacheck -go 1.5 -ignore 'github.com/lib/pq/conn_test.go:SA1019' ./...
```

Comment:
Can you describe why the error ignore was able to go away in the commit message?

Reply:
Updated the commit message and the PR description.